src/fact_generator.ts

```diff
@@ -415,6 +416,16 @@ async function recentUniquePlayers(): Promise<string[]> {
     return output;
 }
 
+async function currentUniquePlayers(): Promise<Array<string>> {
```

Comment:
These facts are only refreshed once every 24 hours, the data would be very stale

Reply:
Oh yeah, totally forgot about that. I'll close this PR